backend/app/services/device/local_provider.py

```
@@ -0,0 +1,505 @@
+                            "title": task_crd.spec.title,
+                            "status": (
+                                task_crd.status.status if task_crd.status else "UNKNOWN"
+                            ),
+                            "created_at": (
+                                task_crd.status.createdAt.isoformat()
+                                if task_crd.status and task_crd.status.createdAt
+                                else None
+                            ),
+                        }
+                    )
+                except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)